quarkc/command.py

```diff
@@ -16,9 +16,9 @@
 Quark compiler.
 
 Usage:
-  quark [options] install [ --java | --python | --javascript | --all ] <file>...
-  quark [options] compile [ -o DIR ] [ --java | --python | --javascript | --all ] <file>...
-  quark [options] run ( --java | --python | --javascript ) <file>...
+  quark [options] install [ (--java | --python | --javascript | --ruby)... | --all ] <file>...
+  quark [options] compile [ -o DIR ] [ (--java | --python | --javascript | --ruby)... | --all ] <file>...
```

Comment:
Why `[ (--java | --python | --javascript | --ruby)... | --all ]` and not `[--java | --python | --javascript | --ruby | --all ]`?